addons/strings/fnc_replace.sqf

```diff
@@ -19,35 +19,22 @@ Example:
     (end)
 
 Author:
-    jaynus
+    BaerMitUmlaut
 --------------------------------------------------------------------------- */
 
 #include "script_component.hpp"
 
 SCRIPT(replace);
 
-// ----------------------------------------------------------------------------
+params [["_string", "", [""]], ["_find", "", [""]], ["_replace", "", [""]]];
+private ["_offset", "_index"];
+if (_find == "" || {_replace find _find != -1}) exitWith {_string};
 
-params ["_string","_pattern","_replacement"];
-private["_i", "_cp", "_findIndex", "_stringArray", "_replaceArray", "_returnArray"];
+_offset = count (_find splitString "");

```

Comment:
The point of `_offset` is, to find ASCII (?) characters that all string commands have problems with.

```
`"ö" splitString ""`
-> ["�","�"]
```